index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>Document</title>
+</head>
+
+<body>
+  <header class="header">
+    <nav class="nav-bar">
+      <span class="name"><a href="#">Miguel</a></span>
+      <span class="menu-icon"><a href="#"><img src="images/menu.svg" alt="menu icon">
```

Comment:
[Optional]
- [ ] It's better if you remove the `a` tag around the hamburger menu because it shouldn't be an anchor element (`a` tag) as it will function like a button that opens the mobile menu ✍🏻 however, this way when you click the menu icon it will take you to home instead.